src/components/MeasureResults/MeasureResultsTable.js

```diff
@@ -17,7 +18,7 @@ function generateMeasureRowValues(measureResult) {
   }
 }
 
-function MeasureResultsTable({ currentResults, handleMeasureChange }) {
+function MeasureResultsTable({ currentResults, handleMeasureChange, colorMapping }) {
```

Comment:
You need to add this to your props.

Reply:
I'm not sure what you mean. They are in the props on both ends.

![image](https://user-images.githubusercontent.com/35747608/158269482-848aaef4-2273-4339-a292-e8fd201f5d62.png)

Reply:
Ah, I see. in `D3Props.js.`